### PR TITLE
feat(HACBS-1436): add fbc label check policy

### PR DESCRIPTION
**Pull request by hongweiliu17**
Signed-off-by: Hongwei Liu hongliu@redhat.com

### CODE REVIEW

File: policies/image/fbc_labels.repo [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package fbc_checks
+
+violation_maintainer_required[{"msg": msg, "details":{"name": name, "description": description, "url": url}}] {
+  not input.Labels["operators.operatorframework.io.index.configs.v1"]
+
+  name := "operators_operatorframework_io_index_configs_v1_label_required"
+  msg := "The 'operators.operatorframework.io.index.configs.v1' label should be defined for FBC image"
+  description := "Should set DC-specific label `operators.operatorframework.io.index.configs.v1` for the location of the DC root directory for FBC image."
+  url := "https://source.redhat.com/groups/public/container-build-system/container_build_system_wiki/guide_to_layered_image_build_service_osbs#jive_content_id_Labels"
```
Comment (dirgim) [inline policies/image/fbc_labels.repo:9]: Maybe we can use the `https://docs.openshift.com/container-platform/4.9/operators/admin/olm-managing-custom-catalogs.html#olm-creating-fb-catalog-image_olm-managing-custom-catalogs` as the url here

Comment (hongweiliu17) [reply policies/image/fbc_labels.repo:9]: Thank you! Let me update it.

Comment (dirgim) [reply policies/image/fbc_labels.repo]: Thanks!

**Review by dirgim — APPROVED** [review]
~LGTM~

**Comment by hongweiliu17** [comment]
> @hongweiliu17 I just remembered, can you also add some unit tests for this change to https://github.com/redhat-appstudio/hacbs-test/tree/main/unittests/test_images ?

Thanks for the reminder! I almost forget it..

**Review by dirgim — APPROVED** [review]
LGTM, thanks for adding the unit and integration tests

**Review by jsztuka — APPROVED** [review]
LGTM

**Pull request merged**